Packs/ICalInviteCreation/Playbooks/playbook-Create_and_send_calendar_invite.yml

```diff
@@ -0,0 +1,239 @@
+      name: ""
+      version: -1
+      description: ''
+    taskid: 5223adb7-87e6-46ec-89d1-3ba4b398e151
+    timertriggers: []
+    type: start
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 50
+        }
+      }
+  "1":
+    continueonerrortype: ""
+    id: "1"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "2"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      description:
+        simple: ${inputs.description}
+      end_date:
+        simple: ${inputs.end_date}
+      end_time:
+        simple: ${inputs.end_time}
+      start_date:
+        simple: ${inputs.start_date}
+      start_time:
+        simple: ${inputs.start_time}
+      title:
+        simple: ${inputs.title}
+      uid:
+        simple: ${inputs.uid}
+      url:
+        simple: ${inputs.url}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Creates an iCal event and stores it as a file object within XSOAR
+      id: 4bb9e7e0-1bb3-4699-8237-a1d893374fbc
+      iscommand: false
+      name: Create calendar entry
+      scriptName: iCalEntryCreate
+      type: regular
+      version: -1
+    taskid: 4bb9e7e0-1bb3-4699-8237-a1d893374fbc
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 290
+        }
+      }
+  "2":
+    continueonerrortype: ""
+    id: "2"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "3"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      attachIDs:
+        simple: ${InfoFile.EntryID}
+      body:
+        simple: |-
+          Hello ${inputs.calendar_event_recipient},
+
+          attached please find a calendar entry for a recent task that has been assigned to you in XSOAR.
+
+          You friendly SOC team
+      bodyType:
+        simple: Text
+      subject:
+        simple: XSOAR Event - ${inputs.title}
+      to:
+        simple: ${inputs.calendar_event_recipient}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Send an email.
+      id: 039e366c-3f63-4619-8a32-fefa1162c275
+      iscommand: true
+      name: Send calendar entry
+      script: '|||send-mail'
+      type: regular
+      version: -1
+    taskid: 039e366c-3f63-4619-8a32-fefa1162c275
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 540
+        }
+      }
+  "3":
+    continueonerrortype: ""
+    id: "3"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 3d882c18-cda1-4b04-81c0-a582926de1ee
+      iscommand: false
+      name: Fin.
```

Comment:
This task name is not clear. Usually we will use in the end of the playbook "Done"